src/loom-provider.ts

```diff
@@ -832,17 +987,28 @@ export class LoomProvider {
   }
 
   private _createLogResult(log: EthFilterLog): IEthFilterLog {
+    const removed = log.getRemoved()
+    const blockTime = numberToHexLC(log.getBlockTime())
```

Comment:
Looks like this was a decimal before, won't converting to hex break clients?

Reply:
Fixed, it will only break if the client marks set `loomProvider.strict = true`